go/vt/vtcombo/tablet_map.go

```diff
@@ -179,9 +179,8 @@ func InitTabletMap(ts *topo.Server, tpb *vttestpb.VTTestTopology, mysqld mysqlct
 					if dbname == "" {
 						dbname = fmt.Sprintf("vt_%v_%v", keyspace, shard)
 					}
-					// Copy dbcfgs and override SidecarDBName because there will be one for each db.
-					copydbcfgs := dbcfgs.Copy()
-					copydbcfgs.SidecarDBName.Set("_" + dbname)
+					// Clone dbcfgs and override SidecarDBName because there will be one for each db.
+					copydbcfgs := dbcfgs.Clone()
```

Comment:
Is the cloning required? SidecarDBName setting is removed in the code.

Reply:
It's safer. I also think the DBName will get set differently for each tablet.